src/main/java/core/basesyntax/util/FruitList.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.util;
+
+import core.basesyntax.model.Order;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FruitList {
+    public List<String> createListOfFruit(List<Order> result) {
```

Comment:
I would recommend to create StorageService and move the business logic of working with storage here

Reply:
ok. I will do this.
 but can I push my solution in this form?